gwr/sel_bw.py

```diff
@@ -207,24 +201,117 @@ def search(self, search='golden_section', criterion='AICc', bw_min=0.0,
         self._bw()
 
         return self.bw[0]
+            
+    #hold it for allowing lat-lons in next PR
+    #_haversine formula to calculate distance
+    def _haversine(lat1, lon1, lat2, lon2):
```

Comment:
this should not be defined in an internal method. Also, this should probably use the stuff in `libpysal.cg.sphere`.

Reply:
the `_haversine() ` is not getting called in this PR, just a 'placeholder' for the next PRs which allowing spherical coords, I should have removed it since it is irrelevant. Yeah, I agree, it would be better to use `libpysal.cg.sphere` instead. Thanks for pointing that out.